src/main/java/fi/otavanopisto/kuntaapi/server/index/IndexRemoveService.java

```diff
@@ -3,26 +3,17 @@
 public class IndexRemoveService implements IndexRemove {
 
   private String serviceId;
-  private String language;
 
   @Override
   public String getId() {
-    return String.format("%s_%s", serviceId, language);
+    return serviceId;
   }
 
   @Override
   public String getType() {
     return "service";
   }
   
-  public String getLanguage() {
-    return language;
-  }
-  
-  public void setLanguage(String language) {
-    this.language = language;
-  }
-  
   public String getServiceId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)